docs/web-analytics.md

```diff
@@ -0,0 +1,82 @@
+* Organizations codes - `UNIV_LISBON | UNIV_PORTO`
+* Course code - `COURSE_XPTO`
+* Course tuns titles - `Summer edition | Winter edition`
+* Course runs resource links - `http://example.edx:8073/courses/course-v1:edX+DemoX+Demo_Course/info`
+* Page title - `Introduction to Programming`
+
+## Location of the web analytics javascript
+Use the `WEB_ANALYTICS_LOCATION` settings to decide where do you want to put the Javascript code. Use `head` (**default** value), to put the Javascript on HTML header, or `footer`, to put the Javascript code to the bottom of the body.
+
+## Add a new Web Analytics solution
+
+In this section it's described how you can add support to a different Web Analytics solution.
+
+* change the `web_analytics.html` template
+* define the `WEB_ANALYTICS_ID` setting with your tracking identification
+* define the `WEB_ANALYTICS_PROVIDER` setting with a value that represents your solution, eg. `my-custom-web-analytics-software`
+* optionally change `WEB_ANALYTICS_LOCATION` setting with `head` (default) or `footer` value
+
+- Example of a `web_analytics.html` file customization that prints to the browser console log the dimension keys and values:
+```javascript
+<script type="text/javascript">
+    {% for dimension_key, dimension_value_list in WEB_ANALYTICS.DIMENSIONS.items %}
+        console.log("dimension: index '{{forloop.counter}}' with key '{{ dimension_key }}' with value '{{ dimension_value_list|join:" | " }}'");
+    {% endfor %}
+</script>
+```
+
+Output:
+```
+dimension: index '1' with key 'organizations_codes' with value 'COMPATIBLE-EVEN-KEELED-UTILIZATION-19 | FOCUSED-NEXT-GENERATION-FUNCTIONALITIES-22 | UNIVERSAL-MODULAR-LOCAL-AREA-NETWORK-23'
+dimension: index '2' with key 'course_code' with value '00017'
+dimension: index '3' with key 'course_runs_titles' with value 'Run 0'
+dimension: index '4' with key 'course_runs_resource_links' with value ''
+dimension: index '5' with key 'page_title' with value 'Business-focused zero-defect application'
+```
+
+But you can also contribute to Richie by creating a pull request to add support to a different web analytics solution. In this last case, you have to edit directly the `web_analytics.html` template.
+
+Example of an override of the `web_analytics.html` file:
+```html
+{% block web_analytics %}
+    {% if WEB_ANALYTICS_ID %} 
+        {% if not WEB_ANALYTICS_PROVIDER or WEB_ANALYTICS_PROVIDER == "my_custom_web_analytics_software" %}
+            <script type="text/javascript" src="{% static 'myapp/js/custom_web_analytics_software.js' %}">
+            <script type="text/javascript">
+                // javascript code that startups the custom web analytics software
+            </script>
+        {% endif %}
+    {% endif %}
+{% endblock web_analytics %}
+```
+
+The web analytics dimensions are a dictionary add to the context where each value is a python list, context variable `WEB_ANALYTICS.DIMENSIONS` with the dimensions. Dictionary keys:
```

Comment:
There is something wrong about this line of text?

Reply:
Yes it is. Even me couldn't understand what I've wrote 1 month ago... 
Changed